docs/server.md

```diff
@@ -52,6 +52,45 @@ Select Node 10.x for the environment. Tachyon requires the S3 bucket and region
 
 ![](lambda-env.png)
 
+If you have the need of keeping your S3 bucket private, you can add an Environment Variable, with the key `S3_AUTHENTICATED_REQUEST` and the value `true` (case insensitive). In this case, it will be necessary to create an IAM Role for the function to be able to fetch data from the S3 Bucket.
+
+![](lambda-env-authenticated-request.png)
+
+Here is an example of IAM Role that should be created for this functionality to work. This policy allows the role to access any S3 Bucket with Get and Put permissions, and allows to create logs and log groups as well as Put log events and describe log streams:
+```
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Action": [
+                "s3:GetObject",
+                "s3:PutObject"
+            ],
+            "Resource": [
+                "arn:aws:s3:::*"
```

Comment:
For better practices, I'd include a specific bucket rather than all S3, as I don't think there'd be a need to ever use such an open policy.

Reply:
Makes sense, will update it. Also, this role is already created by `cloudformation-template.json`. Maybe this should be updated as well? Also, if it is already created by the CF template, should it be added in here? 🤔